firebase-config.js

```diff
@@ -18,5 +20,11 @@ firebaseJson.hosting.redirects = parsedRedirects.reduce(
   },
   [],
 );
+firebaseJson.hosting.headers[0].headers.push({
```

Comment:
Can we remove when not in production mode? It'll make the other production stuff easier to remove.

(To be clear, you can test production mode locally. It's just a longer build.)